examples/kernel_approximation/plot_scalable_poly_kernels.py

```diff
@@ -153,11 +175,16 @@
 print(f"Kernel-SVM score on raw features: {ksvm_score:.2f}%")
 
 # %%
+# Comparing the results
+# ---------------------
+#
 # Finally, plot the results of the different methods against their training
 # times. As we can see, the kernelized SVM achieves a higher accuracy,
 # but its training time is much larger and, most importantly, will grow
 # much faster if the number of training samples increases.
 
+import matplotlib.pyplot as plt
+
```

Comment:
Maybe we can add a title to the plot below. Something similar to "Accuracy vs training time trade-off", what do you think?

Reply:
I am not the biggest fan of plot titles (generally) because I find the information they add is often redundant and they tend to harm the data-ink ratio. Perhaps a caption could be added to briefly summarize the figure, but I don't believe it's strictly necessary here.

If there is some merit to adding a title that I have overlooked, or if you think adding a caption would improve the figure, I am open to making a revision here.

Reply:
I don't have a strong opinion about adding a title, as I agree the information is already there. Thanks @jsilke!